### PR TITLE
The navigation bar

### PR DESCRIPTION
**Pull request by BunnyPrince**
Need the nav bar to start creating the activity page

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -59,12 +59,14 @@ dependencies {
     implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
     implementation 'androidx.navigation:navigation-fragment-ktx:2.4.1'
     implementation 'androidx.navigation:navigation-ui-ktx:2.4.1'
+    //implementation 'androidx.test:monitor:1.5.0'
```
Comment (AC159) [inline app/build.gradle:62]: Remove this line please

File: app/src/androidTest/java/com/soen490chrysalis/papilio/LoginActivityUITest.kt [hunk 2]
```diff
@@ -43,6 +43,7 @@ class LoginActivityUITest {
         Espresso.onView(ViewMatchers.withText(R.string.sign_up)).check(ViewAssertions.matches(ViewMatchers.isClickable()))
     }
 
+
```
Comment (AC159) [inline app/src/androidTest/java/com/soen490chrysalis/papilio/LoginActivityUITest.kt:46]: Same thing here

File: app/src/main/java/com/soen490chrysalis/papilio/view/InitialActivity.kt [hunk 3]
```diff
@@ -2,6 +2,7 @@ package com.soen490chrysalis.papilio.view
 
 import android.content.Intent
 import android.os.Bundle
+import android.view.View
```
Comment (AC159) [inline app/src/main/java/com/soen490chrysalis/papilio/view/InitialActivity.kt:5]: Is this import needed?

File: app/src/main/java/com/soen490chrysalis/papilio/view/InitialActivity.kt [hunk 4]
```diff
@@ -24,6 +25,8 @@ class InitialActivity : AppCompatActivity() {
             startActivity(intent)
         }
 
+
+
```
Comment (AC159) [inline app/src/main/java/com/soen490chrysalis/papilio/view/InitialActivity.kt:29]: Remove these lines

**Review by AC159 — CHANGES_REQUESTED** [review]
See comments

**Comment by AC159** [comment]
Is there any progress on this side?

**Comment by DrakeHunterK7** [comment]
> Is there any progress on this side?

Will be done by today end or tomorrow latest

**Pull request closed without merging**